selene/src/opts.rs

```diff
@@ -1,15 +1,34 @@
-use std::{ffi::OsString, path::PathBuf};
+use std::{ffi::OsString, path::PathBuf, str::FromStr};
 
 use structopt::{clap::arg_enum, StructOpt};
 
+#[derive(Clone, Debug)]
+pub struct Patterns {
+    pub vec: Vec<String>,
+}
+
+impl FromStr for Patterns {
+    type Err = Box<dyn std::error::Error>;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        Ok(Patterns {
+            vec: s.split('\0').map(|x| x.to_string()).collect(),
+        })
+    }
+}
+
 #[derive(Clone, Debug, StructOpt)]
 #[structopt(rename_all = "kebab-case")]
 #[structopt(setting(structopt::clap::AppSettings::ArgsNegateSubcommands))]
 #[structopt(setting(structopt::clap::AppSettings::SubcommandsNegateReqs))]
 pub struct Options {
     /// A glob to match files with to check
-    #[structopt(long, default_value = "**/*.lua")]
-    pub pattern: String,
+    #[cfg_attr(
+        feature = "roblox",
+        structopt(long, default_value = "**/*.lua\0**/*.luau")

```

Comment:
It seems like `\0` and `/` are the only universally prohibited characters in filenames

Reply:
This is just for the default value, though. Users would just pass `--pattern` multiple times